scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
If there is no bracket provided, there anyway does not exist any interval, and hence the whole statement itself does not make much sense.

We could leave it in its current form, or make this fact clearer. I don't think we need to though, but please let me know if anyone else feels otherwise.